apps/src/applab/DesignModeBox.jsx

```diff
@@ -1,6 +1,5 @@
 /* global $ */
```

Comment:
Agreed that making `$` (and maybe `jQuery`) at the Gruntfile/.jshintrc level is a good idea at this point.  I think declaring jQuery at a file-level was useful back when we thought we might remove it.